composer.json

```diff
@@ -13,7 +13,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "ext-gd": "*"
+        "ext-gd": "*",
+        "ext-pdo": "*"
```

Comment:
This is also in suggest section. And not every project that uses securimage require the database storage, and for them it is an unnecessary dependency.

Reply:
Fixed